core/src/test/resources/workspace.properties

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2017 Adobe. All rights reserved.
+# This file is licensed to you under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+# OF ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+#
+# meta_scopes : comma separated list of metascopes associated with your API
+# sample meta_scopes: /s/ent_user_sdk,/s/ent_marketing_sdk,/s/creative_sdk
+meta_scopes=changeMe
+# ims_org_id = your IMS Organization ID as shown in your Adobe Developer Console workspace
+ims_org_id=changeMe
+# consumer_org_id = your consumer Org Id as shown in your Adobe Developer Console workspace
+consumer_org_id=changeMe
+# api_key = your credential API Key (Client ID) as shown in in your Adobe Developer Console workspace
+api_key=changeMe
+# credential_id = your credential id as shown in your Adobe Developer Console workspace
+credential_id=changeMe
```

Comment:
Since Console uses "client_id", we should also use that only.

Reply:
unfortunately, it's not the clientId, it is yet another id, the (jwt) credential.id associated with your workspace (formerly known as the integrationId)

Reply:
got it, thanks!